File: ReleaseNotes.UI/src/services/project/models/release-notes/release-notes-release-note-entry-response-model.ts
import { NoteEntryType } from "../../../../enums/note-entry-type";

export interface ReleaseNotesReleaseNoteEntryResponseModel {
  id: string;
  text: string;
  url: string;
  type: NoteEntryType;
  createdOnUtc: Date;
  releasePublishedOn: Date;
}
 

File: ReleaseNotes.API/ReleaseNotes.API/Services/Project/Models/ReleaseNotes/ReleaseNotesReleaseNoteEntryResponseModel.cs
using ReleaseNotes.API.Enums;

namespace ReleaseNotes.API.Services.Project.Models.ReleaseNotes;

public class ReleaseNotesReleaseNoteEntryResponseModel
{
    public Guid Id { get; set; }
    public string Text { get; set; } = string.Empty;
    public string? Url { get; set; }
    public DateTimeOffset CreatedOnUtc { get; set; }
    public NoteEntryType? Type { get; set; }
    public DateTimeOffset? ReleasePublishedOn { get; set; }
}
